javascript/extractor/lib/typescript/package.json

```
@@ -2,7 +2,7 @@
     "name": "typescript-parser-wrapper",
     "private": true,
     "dependencies": {
-        "typescript": "5.5.2"
+        "typescript": "^5.6.2"
```

```suggestion
        "typescript": "5.6.2"
```

Lets lock it to `5.6.2`.